packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { WORKFLOW_MODEL_UID, STAGE_MODEL_UID } = require('../../constants/workflows');
+
+module.exports = ({ strapi }) => ({
+  async bootstrap() {
+    const wfCount = await strapi.entityService.count(WORKFLOW_MODEL_UID);
+    const stagesCount = await strapi.entityService.count(STAGE_MODEL_UID);
+
+    if (wfCount + stagesCount === 0) {
+      const defaultStages = require('../../constants/default-stages.json');
+      const defaultWorkflow = require('../../constants/default-workflow.json');
+
+      await strapi.query('admin::workflow-stage').createMany({ data: defaultStages });
+
```

Comment:
This being the service layer,  it makes sense for it to use the DB layer.

Still, I personally use we should use the entityService whenever we can (when performance is not critial), as it has the stablest interface.

wdyt?